src/components/ObjectInspector.vue

```diff
@@ -291,6 +295,49 @@ export default {
 
     const useWord = computed(() => (object.value?.data?.moveDistance ? "move" : "use"));
 
+    const tapoutText = computed(() => {
+      if (!object.value?.data?.tapoutTrigger) return;
+      const tapoutData = object.value.data.tapoutTrigger;
+      const mode = tapoutData[0];
+      if (mode === 1) {
+        // Tile tapout. Effects a specific tile, so no limit.
+        return `Effected tile: (${tapoutData[1]},${tapoutData[2]})`;
+      };
+      const limit = object.value?.data?.tapoutLimit;
+      const objectWord = limit === 0 || limit > 1 ? "objects" : "object"
+      const limitPhrase = limit > 0 ? `${object.value.data.tapoutLimit} ` + objectWord : objectWord;
+      let locationText = "";
+      if (mode === 0) {
+        // Area tapout
+        locationText = `in a ${tapoutData[1] * 2 + 1}x${tapoutData[2] * 2 + 1} area`;
+      };
+      if (mode === 2) {
+        // Directional tapout. Uusally only 1 direction, but allows for multiple.
+        let dirArray = [];
+        if (tapoutData[1] > 0) dirArray.push(`${tapoutData[1]} ` + (tapoutData[1] > 1 ? "tiles" : "tile") + " North");
+        if (tapoutData[2] > 0) dirArray.push(`${tapoutData[2]} ` + (tapoutData[2] > 1 ? "tiles" : "tile") + " East");
+        if (tapoutData[3] > 0) dirArray.push(`${tapoutData[3]} ` + (tapoutData[3] > 1 ? "tiles" : "tile") + " South");
+        if (tapoutData[4] > 0) dirArray.push(`${tapoutData[4]} ` + (tapoutData[4] > 1 ? "tiles" : "tile") + " West");
+        const directions = dirArray.length < 2 ? dirArray.join(", ") : dirArray.slice(0, -1).join(", ") + " or " + dirArray[dirArray.length - 1];
+        locationText = "up to " + directions;
+      };
+      return "Can effect " + limitPhrase + " " + locationText;
+    });
+
+    const tapoutTip = computed(() => {
+      if (!object.value?.data?.tapoutTrigger) return;
+      if (object.value?.data?.tapoutTrigger[0] === 1) return "Coordinates of effected tile relative to this object";
```

Comment:
At this point, I'm wondering if `tapoutTrigger` should instead be two values, one `tapoutMode` and another `tapoutData`.

Reply:
It's already been referenced a lot, we're assuming the reader has that context, let's make it easy for them (probably us in a years time).